src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6026,16 +6026,19 @@ export class StudyViewPageStore
             this.chartClinicalAttributes.result,
             (acc: { [id: string]: ChartMeta }, attribute) => {
                 const uniqueKey = getUniqueKey(attribute);
-                acc[uniqueKey] = {
-                    displayName: attribute.displayName,
-                    uniqueKey: uniqueKey,
-                    dataType: getChartMetaDataType(uniqueKey),
-                    patientAttribute: attribute.patientAttribute,
-                    description: attribute.description,
-                    priority: getPriorityByClinicalAttribute(attribute),
-                    renderWhenDataChange: false,
-                    clinicalAttribute: attribute,
-                };
+                const priortiy = getPriorityByClinicalAttribute(attribute);
```

Comment:
@haynescd just noticed a little typo in that variable name

Reply:
Hilarious! I cannot spell!!